rnd/autogpt_server/autogpt_server/server/server.py

```diff
@@ -194,144 +213,6 @@ async def websocket_router(self, websocket: WebSocket):
                     await autogpt_server.server.ws_api.handle_unsubscribe(
                         websocket, self.manager, message
                     )
-                elif message.method == Methods.EXECUTION_EVENT:
```

Comment:
Deleted these as don't think its worth the effort to maintain just yet

Reply:
let's do that in a separate PR